frontend/src/components/app/App.js

```diff
@@ -2,7 +2,8 @@ import './App.css';
 import LoginForm from '../auth/LoginForm'
 import SignUpForm from '../user/SignUpForm'
 import React, { useState } from 'react';
-import Feed from '../feed/Feed'
+// import Feed from '../feed/Feed'
+import GamesLobby from '../games_lobby/GamesLobby';
```

Comment:
Is GamesLobby a page component or a component that forms part of the page at URL /lobby?